src/mono/sample/wasm/browser-nextjs/Program.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Sample
+{
+    public class Test
+    {
+        public static void Main(string[] args)
```

Comment:
The `main` method isn't used, is it?

Reply:
The `WasmBuildApp` build target only runs for `Exe`, I removed the other method.